.docker/run.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+# name of image
+IMAGE=lab-website-renderer:latest
+
+# name of running container
+CONTAINER=lab-website-renderer
+
+# choose platform
+PLATFORM=""
+
+# default docker run command
+DOCKER_RUN="docker run"
+
+# fix interactive on windows
+if [[ $OSTYPE == msys* ]]; then
+    DOCKER_RUN="winpty docker run"
+fi
+
+# build docker image
+docker build ${PLATFORM} \
+    --tag ${IMAGE} \
+    --file ./.docker/Dockerfile .
+
+# run built docker image
+${DOCKER_RUN} ${PLATFORM} \
+    --name ${CONTAINER} \
+    --init \
+    --rm \
+    --interactive \
+    --tty \
+    --publish 4000:4000 \
+    --publish 35729:35729 \
+    --volume $PWD:/usr/src/app \
+    ${IMAGE} "$@"
```

Comment:
This version looks like it retains the issue where Docker for Windows gets a Linux-style path and doesn't know what to do with it. I've added a fix based on our conversation the other day, where you found that `cmd //c cd` worked without any modification.

FWIW, it seems that the value of `$OSTYPE` on Windows can vary depending on how you run the bash script. (I found a few values for the variable in this issue: https://github.com/dylanaraps/neofetch/issues/433.) For instance, running it through Git Bash produces `msys`, and through vanilla CYGWIN produces `cygwin`. Unfortunately running through the Windows 10's Windows Subsystem for Linux (WSL), it returns `linux-gnu`, which is the same as other platforms that do use Linux-style paths. I've modified the check for Windows to at least match `msys` and `cygwin`, but it doesn't handle the WSL `linux-gnu` value, which is conflated with real Linux anyway and definitely shouldn't use a Windows path in that case. I think the more robust solution here would be running `docker info` and extracting the platform the Docker daemon is running on, but we can wait on that. For now, I think these changes should be fine.

```suggestion
# default path to current working directory
APP_DIR="$PWD"

# default docker run command
DOCKER_RUN="docker run"

# fix interactive, folder mounting on windows
if [[ $OSTYPE == msys* ]] || [[ $OSTYPE == cygwin* ]]; then
    DOCKER_RUN="winpty docker run"
    APP_DIR="$( cmd //c cd )"
fi

# build docker image
docker build ${PLATFORM} \
    --tag ${IMAGE} \
    --file ./.docker/Dockerfile .

# run built docker image
${DOCKER_RUN} ${PLATFORM} \
    --name ${CONTAINER} \
    --init \
    --rm \
    --interactive \
    --tty \
    --publish 4000:4000 \
    --publish 35729:35729 \
    --volume "${APP_DIR}:/usr/src/app" \
    ${IMAGE} "$@"
```